metadeploy/api/models.py

```diff
@@ -763,7 +777,8 @@ class PreflightResult(models.Model):
     # }
 
     def subscribable_by(self, user):
-        return self.user == user
+        # TODO: Is this what we want?
+        return not self.user or self.user == user
```

Comment:
@wlonk So currently an anonymous user can only `GET` a preflight by knowing the plan.id and having a `uuid` in their session that corresponds to the ScratchOrg that created the preflight. But, this means that _any_ user can subscribe to the WS channel for updates about a preflight, as long as they know the preflight id (which is not hashed). So we probably want to revert this change, and instead add logic to `consumers.py` that checks the session and implements logic similar to the `GET`?